common/src/sbom-diff/format-handlebars-template.js

```diff
@@ -0,0 +1,151 @@
+const handlebars = require('handlebars');
+const marked = require('marked');
+const createDOMPurify = require('dompurify');
+const { JSDOM } = require('jsdom');
+
+// Emoji lookup table: https://www.quackit.com/character_sets/emoji/emoji_v3.0/unicode_emoji_v3.0_characters_all.cfm
+// We are using the unicode emoji codes instead of github emoji codes (e.g. ':warning:'), because 
+// those can also be interpreted by tools outside of github (e.g. browsers)
+const licenseEmojis = {
+  Go: '&#10004;', // heavy check mark
```

Comment:
Not sure if there is a technical reason against (e.g. certain OS or editors not having all the symbols). But I also tend to keep the codes because they are unambiguous, whereas the displayed emoji images differ between environments.